src/context/user.provider.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react"
+import {useState, useEffect} from "react"
```

Comment:
Imports from one package should be in one line.

```suggestion
import React, {useState, useEffect} from "react"
```

Reply:
Done.